boards/xtensa/esp32/esp32-devkitc/src/esp32-devkitc.h

```diff
@@ -245,7 +245,7 @@ int board_ws2812_initialize(
  *
  ****************************************************************************/
 #ifdef CONFIG_ESP32_OPENETH
```

Comment:
shouldn't contain chip specific macro in common.folder

Reply:
I will put header files in the chip specific directory and include them in the common driver.  WDYT?

Reply:
Hello, @xiaoxiang781216 can you please suggest me how to do this?

I thought about moving those definitions to the chip specific `chip.h` and including it from the driver. WDYT?

Or should I create another file like `openeth.h` in each chips directory?

Do you have any other idea? How should I proceed?

Reply:
I have moved the chip specific definitions to `chip.h`, please let me know if there are any other concerns.

Reply:
sorry for late reply, I lose your message.